src/pages/components/notification/usage.mdx

```diff
@@ -4,79 +4,198 @@ description: Notifications are messages that communicate information to the user
+| ------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Inline | Inline notifications show up in task flows, to notify users of the status of an action. They usually appear at the top of the primary content area.              |
+| Toast  | Toasts are non-modal, time-based window elements used to display short messages; they usually appear at the top of the screen and disappear after a few seconds. |
+
+## Formatting
+
+### Anatomy
+
+<Row>
+<Column colLg={8}>
+
+![notification anatomy image](images/notification-anatomy.png)
+
+</Column>
+</Row>
+
+1. **Icon:** Informs users of the type of notification at a glance.
+2. **Title:** Gives users a quick overview of the notification.
+3. **Message (optional):** Provides additional detail and actionable steps for the user to take.
+4. **Action (optional):** Ghost button that allows users to address the notification or navigates them to a page with further details.
+5. **Close button (optional):** Closes the notification.
+
+## Content
+
+Notifications provide limited space for content, and therefore the content must be short and concise. A user should be able to quickly scan the notification, be apprised of the situation, and know what to do next.
+
+### Main elements
 
 #### Title
 
-All notifications have subject titles, which should be short and descriptive. (Example: “Tester-app-02 has crashed.”)
+- The title should be short and descriptive, explaining the most important piece of information.
+- When possible, communicate the main message using just the title.
+- Use a period only if the title is a full sentence.
 
-#### Message
+#### Body content
 
-We recommend the body of the notification be contained within two lines. Be descriptive and include any troubleshooting actions or next steps. When possible, communicate the main message using just the title. You can include [links](/components/link/code) within the notification body that redirect the user to next steps.
+- Be concise and avoid repeating or paraphrasing the title.
+- Limit content to one or two short sentences.
+- Explain how to resolve the issue by including any troubleshooting actions or next steps. You can include links within the notification body that redirect the user to next steps.
 
-#### Dismissal
+#### Action
 
-We recommend that toast notifications automatically disappear after five seconds. Inline notifications are persistent until the user dismisses them. All notifications have at least one method of dismissal (typically, it is a small “x” in the upper right hand corner).
+- Keep labels concise and clearly indicate the action the user can take.
+- Limit action labels to one or two words. For a list of approved action labels, see Carbon's [content guidelines](https://www.carbondesignsystem.com/guidelines/content/action-labels).
```

Comment:
Was surprised to see approved here, is there a process through which teams get labels approved? Or are these recommendations by our team?

Reply:
These labels have been developed through multiple rounds of review with Cloud Data & AI, IoT, and Watson designers and content designers.

Reply:
Suggestion - should read; "For a list of recommended labels...". I'm not sure "approved" is correct here, in that it implies anything else is non-approved (but we don't have any real control over the action labels different teams might use).

Process for choosing which labels are recommended is outside the scope of this PR, but we can discuss if there is something we want to document/clarify.

Reply:
I don't think it's implying that labels have to be approved before using, but merely saying the action labels on that content page are approved to use. Maybe it should say "For a list of _suggested_ action labels, ..."

Reply:
It needs to be stronger than just suggested. If people have concerns with "approved" then I'd like to go with "recommended".

Reply:
oops sorry y'all my internet is terrible and for some reason no one else comments loaded so I wasn't purposefully ignoring any earlier comments.